tools/jit/templates/register_aten_ops.cpp

```diff
@@ -63,9 +63,7 @@ RegisterOperators reg({
   "aten::get_device(Tensor self) -> int",
   [](Stack & stack) {
       autograd::profiler::RecordFunction record("get_device");
-      auto result = at::get_device(
-          (std::move(peek(stack, 0, 1))).toTensor()
-      );
+      auto result = (std::move(peek(stack, 0, 1))).toTensor().device().index();
```

Comment:
@goldsborough I'm pretty sure this `std::move` isn't necessary. I guess we missed this one because it's in templates?

Reply:
(Probably should raise an error for non-CUDA devices?)